apps/site/src/views/Resources/components/ResourceCard/ResourceCard.module.scss

```diff
@@ -0,0 +1,55 @@
+.group {
+	margin-top: 50px;
+	width: 344px;
+	position: relative;
+	box-shadow: 0 6px 5px -2px rgba(gray, 0.5);
+}
+
+.tape {
+	background-image: url("~@/assets/images/tape.svg");
+	width: 238px;
+	height: 60px;
+	position: absolute;
+	top: -5%;
+}
+
+.container {
+	position: relative;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	gap: 10px;
+	height: 400px;
+}
+
+.tag {
+	background-image: url("~@/assets/images/resource-link.svg");
+	background-size: 100% 100%;
+	background-repeat: no-repeat;
+	background-position: left;
+	display: inline-block;
+	padding: 1rem;
+	padding-right: 3rem;
+	margin-left: -6px;
+	margin-bottom: 0.5rem;
+	transition:
+		filter 0.13s ease,
+		transform 0.13s ease,
+		filter 0.13s ease;
+
+	color: black;
+	text-decoration: none;
+	font-weight: bold;
+
+	&:hover {
+		transform: translateX(-5px);
```

Comment:
maybe add an underline on hover?

Reply:
Are you suggesting that we use the underline instead of the drop shadow or use both?

Reply:
both

Reply:
Ok will do this

Reply:
Actually, I'm not sure if this is a good idea because then the open in new window icon won't be underlined.